templates/zilencer/emails/confirm_remoteserver.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html lang="en">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <title>Zulip</title>
+    </head>
+    <body>
+        <table width="80%" style="align:center; max-width:800px" align="center">
+            <tr>
+                <td style="font-size:16px; font-family:Helvetica;">
+                    <p>Hi!</p>
+
+                    <p>
+                        We received a request to register a remote server against this
+                        email address for {{ host_name }}. If you would like to confirm
+                        this change, please click this link:
+                        <br />
+                        <a href="{{ activate_url }}" style="color:#08c">{{ activate_url }}</a>
+                    </p>
+
+                    <p>
+                        {% if verbose_support_offers %}
```

Comment:
similar to above, we can just take the "True" branch of this and drop the conditional